src/orion/plotting/backend_plotly.py

```diff
@@ -549,6 +549,318 @@ def build_frame():
     return fig
 
 
+def parallel_advantage(experiments, with_evc_tree=True):
+    """Plotly implementation of `orion.plotting.lpi`"""
+    compute_average = bool(
+        isinstance(experiments, dict)
+        and isinstance(next(iter(experiments.values())), Iterable)
+    )
+
+    def build_group():
+        """Build dataframes for groups of experiments"""
+        if compute_average:
+            data_frames = dict()
+            for name, group in experiments.items():
+                frames = build_frame(group)
+
+                df = orion.analysis.average(
+                    frames, group_by="n_workers", keys=["objective", "duration"]
+                )
+                df["names"] = [name] * len(df)
+
+                data_frames[name] = df
+            data_frames = pd.concat(data_frames)
+        elif isinstance(experiments, dict):
+            data_frames = build_frame(experiments.values(), list(experiments.keys()))
+        else:
+            data_frames = build_frame(experiments)
+
+        return data_frames
+
+    def build_frame(experiments, names=None):
```

Comment:
If I understand correctly, the main difference with `regrets` is how we compute the x axis? Maybe we could add an option to regrets() to plot the x axis based on a function() call instead of a key (but could still support a key).

Reply:
Oh and there are the templates that can be different. That could be another customizable argument. I'm trying to see how we can limit the additional number of functions that we need to maintain.

Reply:
Done in 060b747. Thanks!